README.md

```diff
@@ -1,9 +1,10 @@
 # lua-func
 
 The Lua interpreter split in a way such that each function is in it's own file. Except for a few
-functions that were very hard to split up, for example the `lundump.c` file. Doing that would not
-give additional CHERI security guarantees. However this would allow for experimenting with 
-different groupings of functions. The main purpose of doing this is to get additional security
+functions that were very hard to split up, for example the `lundump.c` file. Just moving the 
```

Comment:
The "just moving" sentence doesn't really make sense -- as, at least as I understand things, "just moving" is really what this PR is about, right? :) So I'm not sure what point's being made here. It's probably worth us getting this right, so I'm happy to bounce ideas back and forth.

Reply:
Yes, but I am trying to articulate that this isn't enough, there needs to be some other work that is not in the repo, to obtain the final result.

Reply:
@ltratt, any thoughts?

Reply:
Can you propose some new text (even if it's rough)?

Reply:
I think this explains it better, but I am not sure.